app/redux/reducers/CollapsedSettingsReducer.js

```diff
@@ -224,15 +212,53 @@ const CollapsedSettingsReducer = (state = defaultState, action) => {
     }
 };
 
+const createDefaultState = () => {
+    const { height, width } = Dimensions.get('window');
+
+    if (width > 350) {
```

Comment:
- [ ] move 350 check to its own function

Reply:
This is done on another PR/Branch